src/components/graph/search/GraphSearchDialog.tsx

```diff
@@ -139,14 +152,62 @@ const GraphSearchDialog = (props: Props) => {
 				setTimestamp(getTimestampAsNumber(filteredSearchHistory[0].item));
 				closeModal();
 			}
-
 			if (filteredSearchHistory.length !== 1 && !currentSearchResult) {
 				ac.current = new AbortController();
 				onValueChange(id, ac.current);
 			}
 		}
 	}, [submittedId]);
 
+	React.useEffect(() => {
+		const searchChannels = {
+			previous: submittedTimestamp
+				? api.sse.getEventSource({
+						type: 'event',
```

Comment:
are you sure that we don't need to search messages?

Reply:
yes, task says that i need to find nearest event